### ISSUE TITLE
Selective minversion capability

### ISSUE THREAD

**Issue by phmccarty** [clearlinux/swupd-server#89]
`swupd_create_update` supports a "minversion" feature that forces regeneration of update content that last changed prior to a given version.

A variation of this feature would be to only regenerate content that last changed in a selected version. In other words, for build CURRENT_VERSION, instead of regenerating *all* content with last_change < MIN_VERSION, only regenerate content with last_change == MIN_VERSION. The regenerated content will have last_change CURRENT_VERSION, same as the "minversion" feature behavior.

**Comment by matthewrsj** [comment]
Why? In the case of a bad release?

**Comment by matthewrsj** [comment]
Trying to determine if this is an issue we should open against mixer.

**Comment by phmccarty** [comment]
> Why? In the case of a bad release?

Yes, in case there is a bad release in the middle of an update stream of otherwise good releases, we could selectively rebuild files that last changed at that bad version.
